src/minefield/game.tests.cpp

```diff
@@ -1,16 +1,79 @@
 #include <gtest/gtest.h>
 #include <minefield/utils.h>
+#include <minefield/game_states.h>
 
 namespace minefield::game::tests
 {
-TEST(enteringBoardMeasures, should)
+class GameTestSuit : public ::testing::Test
 {
-    unsigned int min = 10;
-    unsigned int max = 15;
-    unsigned int width = Utils::enterNumberInRange("message", 10, 15);
-    bool widthIsBetweenRange = width < min && width > max;
-    EXPECT_TRUE(widthIsBetweenRange);
-}   
+protected:
+    void SetUp() override
+    {
+    }
 
+    void TearDown() override
+    {
+    }
+    
+    void addPlayer(unsigned int remainingMines)
+    {
+        Player player;
+        player.remainingMines = remainingMines;
+        context.players.push_back(player);
+    }
 
+    void setBoard(unsigned int width, unsigned int height) 
+    {
+        context.board;
+        context.width = width;
+        context.height = height;
+        Utils::initializeBoard(context.board, context.width, context.height);
+    }
+
+    GameContext context;
+};
+
+TEST_F(GameTestSuit, should_continue_game_if_two_players_have_mines)
+{
+    setBoard(10, 10);
+    addPlayer(3);
+    addPlayer(3);
+
+    EXPECT_TRUE(GameStates::stateCheckingNextTurn(context).updateFunction == GameStates::statePuttingMines);
```

Comment:
#should use `EXPECT_EQ` instead, `EXPECT_EQ(GameStates::stateCheckingNextTurn(context).updateFunction, GameStates::statePuttingMines);`